tool/formula/src/test/java/org/openscience/cdk/tools/manipulator/MolecularFormulaManipulatorTest.java

```diff
@@ -1346,14 +1346,14 @@ public void testImplicitH() throws Exception {
     @Test
     public void largeNumberOfAtoms() {
         IChemObjectBuilder builder = SilentChemObjectBuilder.getInstance();
-        assertThat(getAtomCount(getMolecularFormula("C2147483647", builder)),
-                is(2147483647));
+        assertThat(getAtomCount(getMolecularFormula("C9999999", builder)),
+                is(9999999));
     }
 
     @Test(expected = NumberFormatException.class)
     public void tooLargeNumberOfAtoms() {
         IChemObjectBuilder builder = SilentChemObjectBuilder.getInstance();
-        assertThat(getAtomCount(getMolecularFormula("C2147483648", builder)),
+        assertThat(getAtomCount(getMolecularFormula("C10000000", builder)),
                 is(2147483647));
```

Comment:
Maybe update the line too to expect 10000000? I know that for the test it won't matter, but will surely not freak out people in the future reading the code :)